charts/buildbuddy-executor/templates/deployment.yaml

```diff
@@ -126,6 +124,8 @@ spec:
                 - name: server-type
                   value: buildbuddy-executor
           volumeMounts:
+            - mountPath: /root/buildbuddy/
```

Comment:
`$HOME` should just be `/root`, right?

Reply:
(to clarify, I think the actual `UserConfigDir` resolves to `/root/.config/buildbuddy`)

Reply:
Yes, but we don't need to mount the entire `$HOME` dir, especially when it contains custom docker config that might be needed to activate the right credential helpers.

So it's better to narrow down the mount to `$HOME/buildbuddy/` which is `/root/buildbuddy/`

Reply:
> (to clarify, I think the actual UserConfigDir resolves to /root/.config and then we want to append /buildbuddy to that. So maybe this could be /root/.config/buildbuddy ?)

Hmm, let me double check.

Reply:
@bduffany you are right. I misread 😓 

I pushed a fix and renamed the mount to be more relevant.